### PR TITLE
Introduce OnPlayClick to track learners' clicks

### PR DESCRIPTION
**Pull request by AlexandrRzz**
https://trello.com/c/MsdYoWdS/16310-nejm-track-learners-clicks-on-the-learning-media-for-analytics?filter=label:Release%202.5
Related PR: https://github.com/area9innovation/innovation/pull/2698

### CODE REVIEW

File: lib/ui/playeroptions.flow [hunk 1]
```diff
@@ -23,4 +23,5 @@ export {
 		NoControls();
 		FullScreenVideo(isFullScreenVideo : DynamicBehaviour<bool>);
 		MobileStyleVideoPLayer(); //use mobile UI style for video
+		OnPlayClick(fn : (string, string) -> void);
```
Comment (ykuzmenko73) [inline lib/ui/playeroptions.flow:26]: add comment about the style and function inside

**Review by ykuzmenko73 — COMMENTED** [review]
LGTM

File: lib/ui/videoplayer.flow [hunk 2]
```diff
@@ -1313,8 +1316,11 @@ makeSoundPlayer(url : string, options : [PlayerOption]) {
 	pausePlayFn = \-> {
 		if (!i2b(getValue(isStopped))) {
 			nextDistinct(isPaused, !getValue(isPaused));
-			if(!getValue(isPaused) && !(^wasPlayed))
+			if(!getValue(isPaused) && !(^wasPlayed)) {
+				onPlayClickFn = extractStruct(options, OnPlayClick(nop2)).fn;
+				onPlayClickFn("WigiMediaAudio", url);
```
Comment (ykuzmenko73) [inline lib/ui/videoplayer.flow:1321]: It looks better introduce union type with sources of clicks intead of strings

**Review by ykuzmenko73 — APPROVED** [review]

**Comment by thnxvlad** [comment]
[Coach stops speaking when video is fullscreen](https://trello.com/c/wkdQsy0C/16497-coach-stops-speaking-when-video-is-fullscreen)

**Pull request merged**